Tests/js/views/services/plugins/assets/ez-searchplugin-tests.js

```diff
@@ -111,7 +147,8 @@ YUI.add('ez-searchplugin-tests', function (Y) {
             this.view.fire('contentSearch', {
                 viewName: this.viewName,
                 search: {
-                    criteria: {},
+                    offset: 42,
```

Comment:
ok I understand, with your patch offset and limit become mandatory. This is a BC break (most likely this breaks some parts of PlatformUI) so the solution is not to fix the unit tests but fix the code.